### PR TITLE
Documentation review and formating

### PR DESCRIPTION
**Pull request by joaoCarvalho1000**

### CODE REVIEW

File: hypergraph/concepts/02-testnet.md [hunk 1]
```diff
@@ -6,12 +6,7 @@ hide_table_of_contents: false
 ---
 <intro-end />
 
-Constellation provides developers with a full featured testnet to test applications and metagraphs before they're ready for the production environment. 
-The testnet has all of the same features as mainnet and can therefore be used to validate application integrations in a realistic way. 
-
-:::info Developing a metagraph
-Metagraph developers may use testnet as they're nearing production readiness. For initial development, developing on your own test cluster is recommended. See [Deploying to Testnet](/sdk/guides/deploy-testnet).
-:::
+Constellation provides developers with the testnet, a  testing environment where new protocol updates are released and tested before they reach production. Unlike the more stable integration net, the testnet is a  platform for developers wanting to experiment with the latest features and validate their applications' compatibility with upcoming changes.
```
Comment (AlexBrandes) [inline hypergraph/concepts/02-testnet.md:9]: Some awkward language here - could use a rewrite.

File: hypergraph/concepts/03-integrationnet.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Integration net
+slug: /integrationnet
+sidebar_label: Integration net
+hide_table_of_contents: false
+---
+<intro-end />
+
+Constellation provides developers with a full featured integration net to test applications and metagraphs before they're ready for the production environment. 
+The integration net has all of the same features as mainnet and can therefore be used to validate application integrations in a realistic way.
```
Comment (AlexBrandes) [inline hypergraph/concepts/03-integrationnet.md:10]: We need a note somewhere in here that indicates that there's a seedlist and how to apply to the seedlist.

File: hypergraph/concepts/03-integrationnet.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Connecting to Integration net
+The following urls can used to access integration net: 
+- __Block Explorer API__: https://be-integrationnet.constellationnetwork.io
+- __Global L0 API__: https://l0-lb-integrationnet.constellationnetwork.io
+- __DAG L1 API__: https://l1-lb-integrationnet.constellationnetwork.io
+
+See [Available APIs](/hypergraph/global-apis) for detailed information on available endpoints. 
+
+## Faucet
+Constellation hosts a faucet which distributes integration net DAG for testing purposes. This coin has no value and can only be used on the integration net. The faucet provides small amounts of DAG at each request with rate limiting to prevent depletion of its DAG reserves.
```
Comment (AlexBrandes) [inline hypergraph/concepts/03-integrationnet.md:25]: This doesn't exist yet so we should remove until it does.

File: hypergraph/guides/DAG4.js/03-connecting.md [hunk 4]
```diff
@@ -41,6 +41,21 @@ dag4.account.connect({
 });
 ```
 
+#### 2.0 Integration net
+- Block Explorer API: https://be-integrationnet.constellationnetwork.io
+- L0 API: http://lb-integrationnet.constellationnetwork.io:9000
+- L1 API: http://lb-integrationnet.constellationnetwork.io:9010
```
Comment (AlexBrandes) [inline hypergraph/guides/DAG4.js/03-connecting.md:47]: These and the testnet urls above should be updated to use the https versions:
```
https://l0-lb-integrationnet.constellationnetwork.io
https://l1-lb-integrationnet.constellationnetwork.io
```

File: learn/advanced-concepts/03-DAG-Token.md [hunk 5]
```diff
@@ -4,63 +4,43 @@ sidebar_label: DAG Token
 hide_table_of_contents: false
 ---
 
-The Hypergraph's native cryptocurrency is known as $DAG. The $DAG operates on an exclusive metagraph that facilitates seamless interaction between metagraphs and network nodes. In other words, $DAG binds network components together and is how the Hypergraph provides utility.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-## DAG L1 Network​
-DAG Layer 1 (L1) is a special case metagraph that validates DAG currency transactions for the network and bundles them into blocks which are then submitted to the global L0 network for final validation. This metagraph enables $DAG which is the native currency of the network
+##
```
Comment (AlexBrandes) [inline learn/advanced-concepts/03-DAG-Token.md:10]: Is this supposed to have a heading?

File: learn/advanced-concepts/03-DAG-Token.md [hunk 6]
```diff
@@ -4,63 +4,43 @@ sidebar_label: DAG Token
...
 
-The DAG L1 network is a special case metagraph that validates transactions, runs them through parallel rounds of consensus, and processes them into blocks that are then submitted to the Global L0 network for inclusion in a Global Snapshot. DAG is the native currency of the network.
+DAG is the Hypergraph's native cryptocurrency. The [$DAG token](https://constellationnetwork.io/dag-token) operates on an exclusive metagraph that facilitates seamless interaction between metagraphs and network nodes. In other words, $DAG binds network components together and is how the Hypergraph provides utility.
 
-### What is $DAG ?
+The mission of the DAG token, is to promote Constellation's Network by incentivizing participation from all stakeholders, from Validator Nodes to metagraphs.
```
Comment (AlexBrandes) [inline learn/advanced-concepts/03-DAG-Token.md:14]: The DAG L1 isn't a metagraph and it doesn't really facilitate interaction between metagraphs yet. I would keep the previous text that this replaced but just remove the out-there/extra stuff like "rave reviews", "sought after assets", etc. I would also change "$DAG works as an exclusive dApp" to "DAG operates on a special-case Layer 1 network that interacts directly with the Global L0"

File: learn/advanced-concepts/03-DAG-Token.md [hunk 7]
```diff
@@ -4,63 +4,43 @@ sidebar_label: DAG Token
...
+</DocsCards>
 
 ## Who Uses $DAG?
-
-As opposed to traditional cryptocurrencies such as Bitcoin and Ethereum, Constellation's $DAG coin offers several benefits that can improve crypto transactions' efficiency and security. The architecture of the system is ideal for:
-
-### Token Purchasers and Holders
-
-Individuals that have purchased $DAG, are speculating, trading, or sending $DAG between their wallets, using it as a peer-to-peer currency.
+The DAG token plays an important role in incentivizing key players in Constellation's Network:
```
Comment (AlexBrandes) [inline learn/advanced-concepts/03-DAG-Token.md:27]: I prefer the previous version of this section to the changes. Maybe remove references to "dApp"

File: learn/advanced-concepts/04-metagraphs.md [hunk 8]
```diff
@@ -31,6 +31,6 @@ The goal here is to support this level of customization, by splitting operations
 Metagraphs can define their own tokens, known as metagraph tokens, which can be used to transact within their network, reward node operators, and assign value to data. Learn more about the [Metagraph Token Standard](/metagraphs/metagraph-tokens/overview). 
 
 :::info Want to learn more about building a metagraph? 
-Detailed information on the design and implementation of metagraphs can be found in the [Build a Metagraph](/metagraphs) section.
+Detailed information on the design and implementation of metagraphs can be found in the [Build a metagraph](/metagraphs) section.
```
Comment (AlexBrandes) [inline learn/advanced-concepts/04-metagraphs.md:34]: This is a title so it should be capitalized

File: learn/tools-resources/01-tokenomics-litepaper.md [hunk 9]
```diff
@@ -17,19 +17,19 @@ Implementing Generative Tokenomics in the Hypergraph for Validator Nodes and Met
...
+This document is a high-level overview and framing of Constellation’s Hypergraph Network (HGTP): A decentralized network economy where consensus is run on the Global L0 (Hypergraph) and L1 metagraphs (previously referred to as “state channels”). The intention of this paper is to educate individuals on the fundamentals of HGTP with an explanation of the metrics used across the Hypergraph network to yield an incentivization structure for key stakeholders in the economy: Validator Nodes and metagraphs. Additionally, the aim of this overview is for individuals and businesses to gain a better understanding of how to navigate the Constellation Ecosystem with the goal of attracting them to join one or both of these stakeholder groups.
```
Comment (AlexBrandes) [inline learn/tools-resources/01-tokenomics-litepaper.md:20]: We shouldn't change anything in the litepaper.

File: validate/setup-guides/aws/sg.md [hunk 10]
```diff
@@ -144,7 +144,7 @@ Our inbound rules will look something like this 👇
 **[REPEAT](#constellation-sg-rules)** This process for ports `9010-9011`.
 
 :::info
-Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) API access. Ports 9000 and 9001 are configurable and can be independent (to your needs) on the Hypergraph Global Layer0 or State Channel network you connect to.  Node Operators will need to learn what ports are opened for access to the State Channels, and update their firewall (change) accordingly.
+Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) API access. Ports 9000 and 9001 are configurable and can be independent (to your needs) on the Hypergraph Global Layer0 or Metagraph network you connect to.  Node Operators will need to learn what ports are opened for access to the Metagraphs, and update their firewall (change) accordingly.
```
Comment (AlexBrandes) [inline validate/setup-guides/aws/sg.md:147]: Should be lowercase "metagraph"

File: validate/testnet/intro.md [hunk 11]
```diff
@@ -23,5 +23,5 @@ While on the **TestNet** you will be able to:
 
 - Safely and securely manipulate and test your Validator Node's 
 functionality.
-- Help other State Channel companies build, test and prepare to deploy their State Channels.
+- Help other Metagraph companies build, test and prepare to deploy their Metagraphs.
```
Comment (AlexBrandes) [inline validate/testnet/intro.md:26]: lowercase metagraph

**Review by AlexBrandes — CHANGES_REQUESTED** [review]

File: hypergraph/concepts/02-testnet.md (hunk 1 above)
Comment (joaoCarvalho1000) [reply hypergraph/concepts/02-testnet.md:9]: Metagraph developers can use testnet as an experimental environment for testing metagraphs against the newest protocol updates before they are rolled out to integrationnet. Testnet provides a platform for developers to experiment with the latest features, enabling them to validate how their metagraphs will interact with upcoming changes.

File: hypergraph/concepts/03-integrationnet.md (hunk 2 above)
Comment (joaoCarvalho1000) [reply hypergraph/concepts/03-integrationnet.md:10]: the info section already points out to the setup a metagraph guide, that has that information, do you want to also have in this page?

**Review by AlexBrandes — APPROVED** [review]

**Pull request merged**